server/balancer.go

```diff
@@ -35,7 +35,7 @@ type Balancer interface {
 	ScoreType() scoreType
 }
 
-func selectFromStore(stores []*storeInfo, excluded map[uint64]struct{}, filters []Filter, st scoreType) *storeInfo {
+func selectFromStore(stores []*storeInfo, excluded map[uint64]bool, filters []Filter, st scoreType) *storeInfo {
```

Comment:
Seems no difference,  but usually we use map[uint64]struct{} instead of map[uint64]bool, it is more effective.